packages/govuk-frontend/src/govuk/components/table/_index.scss

```
@@ -9,6 +9,18 @@
     border-collapse: collapse;
   }
 
+  @if $govuk-new-typography-scale {
+    // Modifier for tables with a lot of data. Tables with lots of data benefit
+    // from a smaller font size on small screens.
+    .govuk-table--small-text-until-tablet {
+      @include govuk-font-size($size: 16);
+
+      @include govuk-media-query($from: tablet) {
+        @include govuk-font-size($size: 19);
+      }
+    }
```

Mind if we pair "until-tablet" with our `$until:` argument?

Then we can fall back to the default `$size: 19` above without setting it twice

```suggestion
    .govuk-table--small-text-until-tablet {
      @include govuk-media-query($until: tablet) {
        @include govuk-font-size($size: 16);
      }
    }
```